index.bs

```diff
@@ -1012,6 +1032,33 @@ The {{ScopedCredentialEntity}} dictionary describes a user account or a [=relyin
         a user's avatar or a [=relying party=]'s logo.
 </div>
 
+
+### Authenticator Selection Criteria ### {#authenticatorSelection}
+
+The {{AuthenticatorSelectionCriteria}} dictionary describes a set of criterias for which the [RPS] can use to find the appropriate
+authenticator 
+
+<xmp class="idl">
+    dictionary AuthenticatorSelectionCriteria {
+        Attachment    attachment;
+        boolean       requireResidentKey;
```

Comment:
> If the Authenticator is roaming the key is roaming (by definition of Authenticator).

We need to refine the webauthn definition for [=authenticator=] to clarify this. also, there is not a dfn for a "roaming credential private key" (as yet).